.travis.yml

```diff
@@ -0,0 +1,30 @@
+# This is how to tell Travis to use the fast container-based test
+# runner instead of the slow VM-based one.
+sudo: false
+
+language: go
+
+# Only the last two Go releases are supported by the Go team with security updates.
+# Any older versions are considered deprecated.
+go:
+  - "1.10.x"
+  - "1.9.x"
+
+# Only clone the most recent commit.
+git:
+  depth: 1
+
+go_import_path: github.com/aws/amazon-vpc-cni-k8s
+
+# Install needed tools
+install:
+  - go get -u github.com/golang/dep/cmd/dep
+  - go get -u golang.org/x/lint/golint
+  - dep ensure
```

Comment:
You should not need to run dep ensure as part of the build.  All of the dependencies should be in the vendor directory already and properly versioned.  If they are not the build should fail.

Reply:
You are right! It did fail at first on my mac, but that was because of OS X. Removing the dep stuff.

Reply:
Passed without dep ensure: https://travis-ci.org/mogren/amazon-vpc-cni-k8s/builds/396655872